collections/Lat_th/MS_Lat_th_e_22.xml

```diff
@@ -51,132 +46,176 @@
-                  </origin>
-                  <provenance>'Iste liber est <orgName key="org_415145858126223022295"><!-- check? -->
-                        cartusien. prope Erfford
-                     </orgName>', 15th cent. (fol. 1)</provenance>
-                  <provenance>Perhaps
-                     in the <orgName key="org_415145858126223022295" cert="low">
-                        Salvatorberg charterhouse
-                     </orgName> catalogue, 15th century (as 0 38 or 39) (P. Lehmann, <title>Mittelalt. Bibliothekakataloge Deutschlands</title> II, 496)</provenance>
-                  <provenance>Bought 1935.</provenance>
+                     <origPlace><country key="place_7000084">German</country>, <settlement key="place_7024031">Erfurt</settlement>, <orgName key="org_415145858126223022295">Charterhouse of Salvatorberg</orgName></origPlace>: <quote>Iste liber est cartusien’ prope Erfford’</quote> (fol. 1r, 15th century); <quote>est Carthus’ prope Erffordiam</quote> (fol. 1r, 15th century); <quote>Est Carthusien’ prope Erffordiam</quote> (text hand, fol. 132v). Probably 038 or 39 in the catalogue by P. Lehmann, <title>Mittelalt. Bibliothekakataloge Deutschlands</title> II, 496.</origin>
+                  <provenance>Label <q>77</q> (19th century) on spine.</provenance>
+                  <acquisition>Purchased from <orgName key="org_154074582" role="fmo">Bernard Halliday</orgName> booksellers, December 1935 (fol. 1r).</acquisition>
                </history>
                <additional>
                   <adminInfo>
                      <recordHist>
-                        <source>Typescript description by Bodleian Library staff</source>
+                        <source>Revised from typescript description by Bodleian Library staff.</source>
                      </recordHist>
                   </adminInfo>
                </additional>
+               <msPart>
+                  <msIdentifier>
+                     <altIdentifier type="partial">
+                        <idno type="part">MS. Canon. Pat. Lat. 52 (binding fragments)</idno>
+                     </altIdentifier>
+                  </msIdentifier>
+                  <msContents>
+                     <textLang mainLang="la">Latin</textLang>
+                     <msItem>
+                        <locus>(upper pastedown; fol. 133r–v</locus>
+                        <author key="person_58608059">Pope Eutychianus</author>
+                        <title>Exhortatio ad presbyteros</title>
+                        <note>Fragment; PL 5, col. 163–167B, ‘propter quosdam infideles’.</note>
+                     </msItem>
+                     <msItem>
+                        <locus>(fol. 133v)</locus>
+                        <author key="person_111045814">Burchard of Worms</author>
+                        <title>Corrector Burchardi</title>
```

Comment:
Looks like an authority record needs to be created for this and `@key` attribute added

Reply:
In VIAF as [261384780](https://viaf.org/viaf/261384780/).

Reply:
= work_6464